app-crypt/libu2f_host/libu2f_host-1.1.10.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="Yubico Universal 2nd Factor (U2F) Host C Library"
+DESCRIPTION="libu2fhost provides a C library and command-line tool \
+that implements the host-side of the U2F protocol. \
```

Comment:
Remove the backslash at the end of line 3, that will start line 4 on a new line in HaikuDepot